rust/bundle-builder/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+authors = ["Stackable GmbH <info@stackable.de>"]
+description = "Stackable Bundle Builder for the OPA"
+edition = "2021"
+license = "OSL-3.0"
+name = "stackable-opa-bundle-builder"
+repository = "https://github.com/stackabletech/opa-operator"
+version = "0.9.0-nightly"
+
+[dependencies]
+stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.13.0" }
+
+futures = { version = "0.3", features = ["compat"] }
+semver = "1.0"
+snafu = "0.7"
+strum = { version = "0.24", features = ["derive"] }
+tokio = { version = "1.17", features = ["full"] }
+tracing = "0.1"
+pin-project = "1.0"
+warp = "0.3"
+tar = "0.4.38"
+flate2 = "1.0"
+
+[dev-dependencies]
+tempdir = "0.3"
+tokio-test = "0.4.2"
```

Comment:
For consistency i would only go for major + minor versions here

Reply:
Done